translation.json

```diff
@@ -0,0 +1,83 @@
+{
+    "widgets": {
+        "formBuilder": {
+            "dataSource": {
+                "loading": "Loading...",
+                "sending": {
+                    "requiredFields": "* Required fields"
+                },
+                "email": {
+                    "invalid": "The input is not a valid email address."
+                },
+                "file": {
+                    "uploaded": "Uploaded:",
```

Comment:
I think the colon should not be part of the translation string

Reply:
Agreed